.python-version

```diff
@@ -1 +1 @@
-3.8.6
\ No newline at end of file
+3.11
```

Comment:
:ok_hand: it's nice that we are not locking down to a specific patch version. This will capture whatever 3.11.x version is availble.